src/app/components/book/book.component.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { BookMock } from 'src/app/helpers/mocks/books.mock';
+
+import { BookComponent } from './book.component';
+
+describe('BookComponent', () => {
+  let component: BookComponent;
+  let fixture: ComponentFixture<BookComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ BookComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(BookComponent);
+    component = fixture.componentInstance;
+    component.book = BookMock;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
Add the respective tests

Reply:
You could test the view in this component, the book that you add in the input shows well in the html